### ISSUE TITLE
May I get access to dataset?

### ISSUE THREAD

**Issue by jasmine789** [chaosallen/IPN_tensorflow#1]
Hi! I found a link on the IEEE data-port(https://ieee-dataport.org/open-access/octa) and I think that this may belong to this repo. However, I can't download the dataset. So, just wonder that is there any method to get access to the raw training and testing OCTA dataset? (I also send a private message to IEEE dataset author and can't get a reply).

Thanks a lot for your help in advance.

**Comment by chaosallen** [comment]
Yes, I am the administrator of this database. I am sorry that this database cannot be made public for the time being. The database is still under maintenance, which is a supplement to the data and ground truth. Currently the database contains only 300 OCTA volume data with 6MM * 6MM field. We plan to expand some 300MM * 300MM field volume data and the corresponding capillary ground truth .


We will make this database public after maintenance. Sorry again, due to the confidentiality agreement of the medical institution on the data, we are temporarily unable to provide the database.



This will wait at least a few months, because the ground truth drawing is time consuming.


Best.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"jasmine789"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月23日(星期一) 上午10:10
收件人:&nbsp;"chaosallen/IPN_tensorflow"<IPN_tensorflow@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;[chaosallen/IPN_tensorflow] May I get access to dataset? (#1)





 
Hi! I found a link on the IEEE data-port(https://ieee-dataport.org/open-access/octa) and I think that this may belong to this repo. However, I can't download the dataset. So, just wonder that is there any method to get access to the raw training and testing OCTA dataset? (I also send a private message to IEEE dataset author and can't get a reply).
 
Thanks a lot for your help in advance.

**Comment by jasmine789** [comment]
Thanks a lot for your reply and wish you all the best!

**Comment by RunningDongxu** [comment]
Hi, thank you for your excellent work and dataset, could I please ask when will you release the password for the dataset?

**Comment by latexers** [comment]
I have send an email according to the format on the IEEE data-port, how long need to get reply ? Thank you.
